tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -333,6 +336,10 @@ func gatherInfo(runner *kubectlcmd.Runner, config *config.BugReportConfig, resou
 	runAnalyze(config, params, analyzeTimeout)
 }
 
+func isZtunnelPod(podName string) bool {
+	return strings.HasPrefix(podName, "ztunnel")
```

Comment:
seeing this exact check getting duplicated across PR's https://github.com/istio/istio/pull/43881 and https://github.com/istio/istio/blob/master/istioctl/cmd/proxyconfig.go#L965
would be good if these could all use the same check

Reply:
@howardjohn `app=ztunnel` pod label?

Reply:
I'll do this in another PR, maybe we can do a better check if we can? If we can't, then we just fallback to the name.